src/main/java/com/onlinebookstore/dto/order/UpdateOrderRequestDto.java

```diff
@@ -0,0 +1,12 @@
+package com.onlinebookstore.dto.order;
+
+import com.onlinebookstore.model.Status;
+import com.onlinebookstore.validation.StatusSubset;
+import jakarta.validation.constraints.NotNull;
+
+public record UpdateOrderRequestDto(
+        @NotNull
```

Comment:
```suggestion
        @NotBlank
```

Reply:
Here the error is thrown by validator on @NotBlank validation over enum, it can't apply it for enum, so I used custom validation and @NotNull. Please, don't consider this as a mistake.